templates/python-scrapy/src/spiders/title.py

```diff
@@ -15,7 +15,7 @@ class TitleSpider(Spider):
     """
 
     name = 'title_spider'
-
+    start_urls = ['https://apify.com']
```

Comment:
This will duplicate the `start_urls` from input, is it needed here?

Reply:
Actually, it won't. However, I'll remove it to make the Spider as simple as possible.